e2e/cypress/tests/integration/archived_channel/archive_channel_add_reaction_spec.ts

```diff
@@ -10,15 +10,17 @@
 // Stage: @prod
 // Group: @channel
 
+import {AdminConfig, TeamSettings} from '@mattermost/types/config';
+
 import * as TIMEOUTS from '../../fixtures/timeouts';
 
 describe('Archived channels', () => {
     before(() => {
         cy.apiUpdateConfig({
             TeamSettings: {
                 ExperimentalViewArchivedChannels: true,
-            },
-        });
+            } as TeamSettings,
+        } as AdminConfig);
```

Comment:
Likely this is not necessary once update to latest master.

Reply:
OK, done